kubernetes-response-engine/deployment/falco/falco-daemonset.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: extensions/v1beta1
```

Comment:
Same here--does this differ from the existing falco daemonset configs already at examples/k8s-using-daemonset ?

Reply:
Same than above, when the new Falco Helm Chart is merged this change will dissapear.

Thanks!